components/pdfviewer/annotations.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Annotations
+page_title: PdfViewer - Annotations
+description: Annotations in the PDF Viewer for Blazor. How to add and configure annotations in the PDF Viewer for Blazor
+slug: pdfviewer-annotations
+tags: telerik,blazor,pdf,pdfviewer,annotations
+published: True
+position: 15
+---
+
+# PdfViewer Annotations
+
+The PdfViewer supports a set of annotation types that the user can add to enhance the PDF document - highlight important text areas or add notes as free text. 
+
+The PDFViewer creates a dedicated layer for the annotations and includes them when downloading or printing the file. When downloading a file with annotations and opening it again, the PDFViewer allows editing the previously created annotations.
```

Comment:
This sentence can be confusing and requires one to read it a couple of times. It's not immediately clear from where the file is downloaded (the PDF Viewer or somewhere else) and where the file is opened (in the PDF Viewer or in the local file system) ?

Reply:
Updated in https://github.com/telerik/blazor-docs/pull/2429/commits/b7bfc5be783e7dede3ea4c742e0a20390182fddd